apps/transformers/lib/transformations/date_time.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Transformers.DateTime do
+  @behaviour Transformation
+
+  alias Transformations.FieldFetcher
+
+  @impl Transformation
+
+  def transform(payload, parameters) do
+    with {:ok, sourceField} <- FieldFetcher.fetch_parameter(parameters, :sourceField),
+         {:ok, sourceFormat} <- FieldFetcher.fetch_parameter(parameters, :sourceFormat),
+         {:ok, targetField} <- FieldFetcher.fetch_parameter(parameters, :targetField),
+         {:ok, targetFormat} <- FieldFetcher.fetch_parameter(parameters, :targetFormat),
+         {:ok, payloadSourceValue} <- FieldFetcher.fetch_value(payload, sourceField),
+         {:ok, sourceDatetime} <- parseTime(payloadSourceValue, sourceFormat, sourceField),
+         {:ok, transformedDatetime} <- formatDateTime(sourceDatetime, targetFormat) do
+      {:ok, payload |> Map.put(targetField, transformedDatetime)}
+    else
+      {:error, reason} ->
+        {:error, reason}
+
+      nil_payload ->
+        {:ok, nil_payload}
+    end
+  end
+
+  defp parseTime(dateString, dateFormat, sourceField) do
```

Comment:
This function is called `parseTime`, but it accepts a `dateString` and a `dateFormat`. Should it be renamed `parseDate`?

Reply:
good call 👍  changed